src/shared/codelens/codeLensUtils.ts

```diff
@@ -132,18 +132,41 @@ function makeConfigureCodeLens({
     return new vscode.CodeLens(range, command)
 }
 
-export function getMetricDatum({ command, isDebug, runtime }: {
+export function getMetricDatum({
+    command,
+    isDebug,
+    runtime,
+    samVersion,
+    duration,
+    attempts
+}: {
     command: string,
     isDebug: boolean,
     runtime: string,
+    samVersion?: string,
+    duration?: number,
+    attempts?: number
+
 }): { datum: Datum } {
+    const metadata = new Map([
+        ['runtime', runtime],
+        ['debug', `${isDebug}`]
+    ])
+    // TODO: Capture this information (Jetbrains captures this)
```

Comment:
Suggest to avoid adding unused parameters/code. We can add the parameter when we capture this information.

Reply:
The latest rework will add an unused type (`SamInvokeStatistics.run`) in order to justify sending a wrapper object from some functions. This will be used by https://github.com/aws/aws-toolkit-vscode/issues/646 .